### DISCUSSION TITLE
How to: Extended Sync Question

### DISCUSSION THREAD

**Discussion by benwestphal** [eclipse-tractusx/tractusx-edc#1057] [Q&A]
Open house meeting to support with anything EDC related. You have questions about EDC, problems running/ configuring EDC in your environment or want to know what´s next in EDC and when to expect? This is the place to ask all these questions.

When: Every Tuesday 10.30 - 11.00 CET (UTC+1)
Where: [Join Teams Meeting here](https://teams.microsoft.com/l/meetup-join/19%3ameeting_Y2JmZWMzNDktNmFkMC00ZThhLTg2N2QtNDRkYmUwNDcwNWRm%40thread.v2/0?context=%7b%22Tid%22%3a%229652d7c2-1ccf-4940-8151-4a92bd474ed0%22%2c%22Oid%22%3a%22a92c6916-f523-4c1d-9ab7-61b5e6eef3be%22%7d)

Get in contact with benjamin.bw.westphal@mercedes-benz.com for a permanent invitation

How to post your question to ensure the highest chances of getting an answer:

1. Start a new discussion
2. Select the Q&A category
3. Start the Title with: [Extended Sync]... your title here

Specifically point 3 is important as it enables us to run a search filter to find your questions!

**Comment by lgblaumeiser** [comment]
Closed as stale, please renew item if this is still an issue
